exercises/practice/binary-search/.approaches/loop-with-switch/content.md

```diff
@@ -8,13 +8,13 @@ package binarysearch
 func SearchInts(list []int, key int) int {
 	for left, right := 0, len(list); left != right; {
 		mid := (left + right) / 2
-    
+
```

Comment:
Sorry about stripping this—I didn't realize I did that until after I submitted. Let me know if I should put it back in to avoid extra noise here.

Reply:
I don't mind the extra line here.